soccer/gameplay/GameplayModule.cpp

```diff
@@ -404,18 +410,6 @@ boost::python::object Gameplay::GameplayModule::getRootPlay() {
 boost::python::object Gameplay::GameplayModule::getMainModule() {
     return _mainPyNamespace["main"];
 }
-
-void Gameplay::GameplayModule::sendFieldDimensionsToPython() {
-    PyGILState_STATE state = PyGILState_Ensure();
-    {
-        try {
-            getMainModule().attr("set_field_constants")(
-                Field_Dimensions::Current_Dimensions);
-        } catch (error_already_set) {
-            PyErr_Print();
-            throw new runtime_error(
-                "Error trying to pass field dimensions to python");
-        }
-    }
-    PyGILState_Release(state);
+boost::python::object Gameplay::GameplayModule::getConstantsModule() {
```

Comment:
Since we only need to access the constants module one time in the GameplayModule constructor, let's remove the `getConstantsModule()` method and just access the python module in the constructor directly.